src/sentry/profiles/task.py

```diff
@@ -269,22 +271,60 @@ def _prepare_frames_from_profile(profile: Profile) -> Tuple[List[Any], List[Any]
         return (modules, stacktraces)
 
 
+def __symbolicate(
```

Comment:
Can we come up with a better name for this? Perhaps `run_symbolication` or even just inline this block